### PR TITLE
Added TensorFlow example.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

File: tensorflow/1.10/README.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+
+<a name="managing"></a>
+# Managing
+
+<a name="updating-configuration"></a>
+## Updating Configuration
+You can make changes to the service after it has been launched.
+Configuration management is handled by the scheduler process, which in turn handles deploying TensorFlow itself.
+
+Edit the runtime environment of the scheduler to make configuration changes.
```
Comment (triclambert) [inline tensorflow/1.10/README.md]: Shouldn't we link out to the TF docs for this?  I presume those capture the difference between updating in Open and Enterprise?

File: tensorflow/1.10/README.md [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+After making a change, the scheduler will be restarted and automatically deploy any detected changes to the service, one node at a time.
+For example, a given change will first be applied to `worker-0`, then `worker-1`, and so on.
+
+Some changes, such as decreasing the number of nodes or changing volume requirements, are not supported after initial deployment.
+See [Limitations](#limitations).
```
Comment (triclambert) [inline tensorflow/1.10/README.md]: This should probably be a link over to the TF docs as well.  However, that section seems to be missing...

https://github.com/mesosphere/dcos-tensorflow/blob/master/docs/index.md#limitations

/cc @elezar

File: tensorflow/uninstall/README.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+enterprise: 'no'
+---
+
+<!-- THIS CONTENT DUPLICATES THE DC/OS OPERATION GUIDE -->
+
+### DC/OS 1.10
+
+If you are running TensorFlow on DC/OS 1.10:
+
+1. Uninstall the service. From the DC/OS CLI, enter `dcos package uninstall --app-id=<instancename> tensorflow`.
```
Comment (triclambert) [inline tensorflow/uninstall/README.md]: Shouldn't this be "beta-tensorflow"?

Comment (elezar) [reply tensorflow/uninstall/README.md]: Yes, that should be `beta-tensorflow`.

**Comment by elezar** [comment]
@joerg84 following @triclambert's comment, it might be a good idea look at this doc from the point of view of someone just wanting a stable example to run. (is that the point of the `examples` repo?)

I think we may have been abusing it as a location for our docs, but the plan is to have https://mesosphere.github.io/dcos-tensorflow/ built off the repo directly and have documentation mirroring our other service docs (this is also tracked in TF-7).

**Pull request merged**